src/components/Accounts/ViewFeesFines/ViewFeesFines.js

```diff
@@ -190,6 +190,12 @@ class ViewFeesFines extends React.Component {
     return res;
   }
 
+  formatTitle(item) {
+    const instanceTypeString = item.materialType ? `(${item.materialType})` : '';
+
+    return `${item.title} ${instanceTypeString}` || '-';
```

Comment:
@annamelnyk I think `|| '-'` would never happen here right? Perhaps we should remove it or refactor it a bit if we do expect the title to be empty.